<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>校园导游系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/Image/whu.ico</normaloff>:/icons/Image/whu.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>549</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>200</y>
          <width>200</width>
          <height>100</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string>导游系统</string>
        </property>
       </widget>
       <widget class="QPushButton" name="enterTourist">
        <property name="geometry">
         <rect>
          <x>332</x>
          <y>368</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>进入游客系统</string>
        </property>
       </widget>
       <widget class="QPushButton" name="enterSecurity">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>368</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>进入保卫部系统</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>0</y>
          <width>469</width>
          <height>700</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/imgs/Image/routegraph.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="changeMap">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>168</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>修改地图</string>
        </property>
       </widget>
       <widget class="QPushButton" name="exitSecurity">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>432</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>退出保卫部系统</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QPushButton" name="exitTourist">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>540</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>退出游客系统</string>
        </property>
       </widget>
       <widget class="QLabel" name="routegraph">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>0</y>
          <width>469</width>
          <height>700</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/imgs/Image/routegraph.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="searchButton">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>400</y>
          <width>200</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>查询路径</string>
        </property>
       </widget>
       <widget class="QPushButton" name="gongzhujiao">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>10</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="songyuancaochang">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>50</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="fengyuan">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>160</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="jiuyiercaochang">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>170</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="luojiaguangchang">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>300</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="yingyuan">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>90</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="guiyuancaochang">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>160</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="jisuanjixueyuan">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="luojiashan">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>300</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="leijunkejilou">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>250</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="luojiamen">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>340</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="xintu">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>420</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="xinghutiyuchang">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>420</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="xincao">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>520</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="wanganxiaocaochang">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>500</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="guoruanc3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>580</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="shishe">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>580</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="xinxixuebuyijiao">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>580</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="luoyumen">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>660</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QPushButton" name="nanermen">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>660</y>
          <width>15</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>go</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="noticeTextEdit">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>73</y>
          <width>431</width>
          <height>161</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>40</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;道路信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actiontourists">
   <property name="text">
    <string>游客</string>
   </property>
  </action>
  <action name="actionsecurity">
   <property name="text">
    <string>保卫部</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
